web3/contract.py

```diff
@@ -540,6 +540,38 @@ def find_functions_by_identifier(cls,
                                      ) -> List[Any]:
         raise NotImplementedError("This method should be implemented in the inherited class")
 
+    @staticmethod
+    def get_fallback_function(
+        abi: ABI, w3: 'Web3',
+        function_type: Union[Type['ContractFunction'], Type['AsyncContractFunction']],
+        address: Optional[ChecksumAddress] = None
+    ) -> Union['ContractFunction', 'AsyncContractFunction']:
+        if abi and fallback_func_abi_exists(abi):
+            return function_type.factory(
+                'fallback',
+                w3=w3,
+                contract_abi=abi,
+                address=address,
+                function_identifier=FallbackFn)()
+
+        return cast('ContractFunction', NonExistentFallbackFunction())
```

Comment:
Should this use `function_type` instead of `'ContractFunction'`?

Reply:
good catch @pacrob. I fixed it but had to add the type ignore because mypy didn't like that type as a variable